subprojects/docs/src/docs/release/notes.md

```diff
@@ -87,6 +87,12 @@ TODO - Java lambdas are supported, and unsupported captured values are reported.
 TODO - File collections queried at configuration time are treated as configuration inputs.
 TODO - File system repositories are fully supported including dynamic versions in Maven, Maven local, and Ivy repositories
 
+### JVM
+
+#### Added Support for using Java 20 via Toolchains
+
+Gradle now supports using Java 20 via [Toolchains](userguide/toolchains.html). Running Gradle on Java 20 is not yet supported.
```

Comment:
The guidelines for this are:

- Set the context - what are we even talking about (don’t assume too much about what the user already knows)
- Express the problem or use case in the given context
- Explain how the new release addresses that problem or use case
- Provide build script snippet or screenshot illustrating the new feature, if applicable
- Link to the full documentation for more details 

Anything you can add to this for it to be more relatable?

Reply:
I moved this to the 8.1 release notes Google doc.